controller/src/beerocks/master/tasks/optimal_path_task.cpp

```diff
@@ -329,8 +337,8 @@ void optimal_path_task::work()
         TASK_LOG(DEBUG) << "responsiveness_precentage_11k: " << int(responsiveness_precentage_11k)
                         << "%";
 
-        if (responsiveness_precentage_11k != 100) {
-            TASK_LOG(DEBUG) << "11k measurement request responsiveness is less than 100%";
+        if (responsiveness_precentage_11k < 50) {
```

Comment:
I would go even further and add this explanation here in a comment. Makes reading the code much easier.

Reply:
I wish I had a good explanation for it, but there isn't one. This is an optimization that @kantera800 asked me to do since most of the time we didn't get 100%, and optimal_path stopped, so changing to 50% made the demo work better, but it isn't necessarily "better" because the reason we don't have 100% might be because the station "didn't want" to reply. So our decision might be wrong.
So I wouldn't write it in the commit message nor in a comment (stupid thing to write IMO).
But if you have and a specific comment you want me to add as a comment or to the commit message, please write it down and I'll add it.

Either way, all of the changes in this commit are pretty small and it has been done a part of optimal_path optimization so I preferred doing it all in a single commit. I don't think that separating it will give extra benefit, and there is no point in doing it now.

Reply:
Bottom line - its a HACK we don't know why is working and might not be working at all?
So at least add a comment (and possibly a task) which explains that this is done as part of demo and should be revised in the future.

Reply:
added a comment to the code.

Reply:
Add a comment in the code